### DISCUSSION TITLE
PrintJob doesn't print transparent png (Windows)

### DISCUSSION THREAD

**Discussion by MalacTheLittle** [airsdk/Adobe-Runtime-Support#1398] [Q&A]
Is there any way to send transparent png to PrintJob? I tried to print to PDF and to physical printer with same results.
It should look like this when printed:
![image](https://user-images.githubusercontent.com/56675407/143604200-93a04640-97ef-4401-baaa-7514be50454c.png)

But print result is like this:
![image](https://user-images.githubusercontent.com/56675407/143604032-5aaf750d-12c7-417c-8331-68297da32029.png)

I'm printing MovieClip with font, shapes and png's.

**Comment by ajwfrost** [comment]
Took me a while to spot it .. but I see what you mean, the PNGs are coming out with a white background that's blocking out the guide lines. 

I believe this is a known restriction with vector-based printing, so can you please switch to bitmap-based printing? i.e. provide a `PrintJobOptions` object when you call `PrintJob.addPage()`, and have the `printAsBitmap` value set to `true`?
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/printing/PrintJobOptions.html
> If the content that you're printing includes a bitmap image, set printAsBitmap to true to include any alpha transparency and color effects. If the content does not include bitmap images, print the content in higher quality vector format (the default option).

thanks

**Comment by MalacTheLittle** [comment]
@ajwfrost Thanks for the hint, I missed that one. Now I'm getting fine result. Still, it's a pitty for that restriction :/ It's always better result to print text as vector-based than raster-based.

![image](https://user-images.githubusercontent.com/56675407/144080067-d0ac2276-8bde-4c60-aa9e-9dd489f49d1a.png)
